Libraries and headers from ffmpeg are installed in usr/lib.

Upstream-Status: Pending
Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

Index: git/Makefile.ffmpeg
===================================================================
--- git.orig/Makefile.ffmpeg	2012-08-26 02:46:40.380518313 +0300
+++ git/Makefile.ffmpeg	2012-08-26 02:46:40.512511177 +0300
@@ -63,5 +63,5 @@
 
 install:
 	cd ffmpeg; make -j9 DESTDIR="$(WORK)/ffmpeg_compiled" install
-	$(HOST)-strip ffmpeg_compiled/usr/local/lib/*.so
+	$(HOST)-strip ffmpeg_compiled/usr/lib/*.so
 
Index: git/Makefile
===================================================================
--- git.orig/Makefile	2012-08-26 02:52:11.578611542 +0300
+++ git/Makefile	2012-08-26 02:52:53.952320465 +0300
@@ -1,7 +1,7 @@
 CFLAGS+=-std=c++0x -DSTANDALONE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -DTARGET_POSIX -D_LINUX -fPIC -DPIC -D_REENTRANT -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -DHAVE_CMAKE_CONFIG -D__VIDEOCORE4__ -U_FORTIFY_SOURCE -Wall -DHAVE_OMXLIB -DUSE_EXTERNAL_FFMPEG  -DHAVE_LIBAVCODEC_AVCODEC_H -DHAVE_LIBAVUTIL_OPT_H -DHAVE_LIBAVUTIL_MEM_H -DHAVE_LIBAVUTIL_AVUTIL_H -DHAVE_LIBAVFORMAT_AVFORMAT_H -DHAVE_LIBAVFILTER_AVFILTER_H -DOMX -DOMX_SKIP64BIT -ftree-vectorize -DUSE_EXTERNAL_OMX -DTARGET_RASPBERRY_PI -DUSE_EXTERNAL_LIBBCM_HOST
 
-LDFLAGS+=-L./ -lc -lWFC -lGLESv2 -lEGL -lbcm_host -lopenmaxil -lfreetype -lz -Lffmpeg_compiled/usr/local/lib/
-INCLUDES+=-I./ -Ilinux -Iffmpeg_compiled/usr/local/include/
+LDFLAGS+=-L./ -lc -lWFC -lGLESv2 -lEGL -lbcm_host -lopenmaxil -lfreetype -lz -Lffmpeg_compiled/usr/lib/
+INCLUDES+=-I./ -Ilinux -Iffmpeg_compiled/usr/include/
 
 SRC=linux/XMemUtils.cpp \
 		utils/log.cpp \
@@ -63,5 +63,5 @@
 	cp omxplayer omxplayer.bin omxplayer-dist/usr/usr/bin
 	cp COPYING omxplayer-dist/usr/share/doc/
 	cp README.md omxplayer-dist/usr/share/doc/README
-	cp -a ffmpeg_compiled/usr/local/lib/*.so* omxplayer-dist/usr/lib/omxplayer/
+	cp -a ffmpeg_compiled/usr/lib/*.so* omxplayer-dist/usr/lib/omxplayer/
 	tar -czf omxplayer-dist.tar.gz omxplayer-dist
